### ISSUE TITLE
Error on plugin startup when solar radiation is enabled

### ISSUE THREAD

**Issue by WotAGitUR** [peledies/Homebridge-Ambient-Weather-Temperature-Sensor#8]
**Describe The Bug:**

During Homebridge and plugin startup this appears in the logs
`This plugin generated a warning from the characteristic 'Current Ambient Light Level': characteristic was supplied illegal value: number 0 exceeded minimum of 0.0001. See https://homebridge.io/w/JtMGR for more info.`

**To Reproduce:**

This only appears when solar radiation is enabled

**Expected behavior:**

Expect not to see error message about zero values, solar radiation is usually zero at night :)

**Logs:**
```
[22/01/2024, 21:10:02] [homebridge-ambient-weather-sensors] This plugin generated a warning from the characteristic 'Current Ambient Light Level': characteristic was supplied illegal value: number 0 exceeded minimum of 0.0001. See https://homebridge.io/w/JtMGR for more info.
[22/01/2024, 21:12:00] [homebridge-ambient-weather-sensors] This plugin generated a warning from the characteristic 'Current Ambient Light Level': characteristic was supplied illegal value: number 0 exceeded minimum of 0.0001. See https://homebridge.io/w/JtMGR for more info.
[22/01/2024, 21:14:02] [homebridge-ambient-weather-sensors] This plugin generated a warning from the characteristic 'Current Ambient Light Level': characteristic was supplied illegal value: number 0 exceeded minimum of 0.0001. See https://homebridge.io/w/JtMGR for more info.
[22/01/2024, 21:16:00] [homebridge-ambient-weather-sensors] This plugin generated a warning from the characteristic 'Current Ambient Light Level': characteristic was supplied illegal value: number 0 exceeded minimum of 0.0001. See https://homebridge.io/w/JtMGR for more info.
[22/01/2024, 21:18:02] [homebridge-ambient-weather-sensors] This plugin generated a warning from the characteristic 'Current Ambient Light Level': characteristic was supplied illegal value: number 0 exceeded minimum of 0.0001. See https://homebridge.io/w/JtMGR for more info.

```

**Plugin Config:**

```
 "name": "Our Weather",
    "apiKey": "xxxxx",
    "applicationKey": "xxxx",
    "temperatureSensors": true,
    "humiditySensors": true,
    "solarRadiationSensors": true,
    "platform": "AmbientWeatherSensors"
```

**Screenshots:**

**Environment:**

* **Plugin Version**: 1.2.1
* **Homebridge Version**: 1.7.0
* **Node.js Version**: v20.9.0
* **NPM Version**: 
* **Operating System**: Raspian>

**Comment by peledies** [comment]
@WotAGitUR thanks for the detailed bug report. 

this issue has been resolved with [v1.2.2](https://github.com/peledies/homebridge-ambient-weather-sensors/releases/tag/v1.2.2)

**Comment by WotAGitUR** [comment]
Thank you @peledies 
Installed that version and no longer see that appearing in the logs :)
